functionary/builder/api/v1/views/build.py

```diff
@@ -1,14 +1,14 @@
-from rest_framework import permissions
-
 from builder.models import Build
+from core.api.permissions import HasEnvironmentPermissionForAction
 from core.api.viewsets import EnvironmentReadOnlyModelViewSet
 
 from ..serializers import BuildSerializer
 
 
 class BuildViewSet(EnvironmentReadOnlyModelViewSet):
+    """View the status of package builds"""
```

Comment:
The description in the swagger docs was defaulting to that of `EnvironmentReadOnlyModelViewSet`, so I added a very basic docstring for now so that at least the generated swagger docs have something more sensible.  We'll surely flesh out the docs down the line.